app/modules/munkiinfo/scripts/munkiinfo.py

```diff
@@ -96,6 +98,26 @@ def get_munkiprotocol():
     except AttributeError:
         return 'Could not obtain protocol'
 
+def middleware_checks():
+    """Check for middleware, get version if supported"""
+
+    middleware_version = []
+    name = []
+    for filename in os.listdir('/usr/local/munki'):
+            print "checking %s" % filename
+            if (filename.startswith('middleware') 
+                and os.path.splitext(filename)[1] == '.py'):
+                
+                name = os.path.splitext(filename)[0]
+                print 'middleware name %s' % name
+                try:
+                    middleware = importlib.import_module(name)
+                    middleware_version = middleware.get_version()
+                except:
+                    print "Error importing middleware for version checks, or version function does not exist."
```

Comment:
This debug statement is of value however also should likely be removed and replaced with a `pass`

To my knowledge @waderobson is the only person in the community that has publicly release middleware for munki. So I'm assuming this `get_version()` method is for your own custom middleware. It might be of value to ping the [s3 auth](https://github.com/waderobson/s3-auth/) & [gcs-auth](https://github.com/waderobson/gcs-auth) middleware and ask Wade to add a `get_version()` method (or possibly create a pull request).